[Console] Document invokable command (#20932)
symfony/symfony-docs: components/console/events.rst

    for these constants to be available.

If you use the Console component inside a Symfony application, commands can

> handle signals themselves. To do so, subscribe to :class:`Symfony\\Component\\Console\\Event\\ConsoleSignalEvent` event::

Reviewer: ```suggestion
handle signals themselves by subscribing to the :class:`Symfony\\Component\\Console\\Event\\ConsoleSignalEvent` event::
```

    // src/Command/MyCommand.php
    namespace App\Command;

    use Symfony\Component\Console\Attribute\AsCommand;
    use Symfony\Component\EventDispatcher\Attribute\AsEventListener;

    #[AsCommand(name: 'app:my-command')]
    class MyCommand
    {
        // ...

        #[AsEventListener(ConsoleSignalEvent::class)]
        public function handleSignal(ConsoleSignalEvent $event): void
        {
            // set here any of the constants defined by PCNTL extension
            if (in_array($event->getHandlingSignal(), [\SIGINT, \SIGTERM], true)) {
                // ...
            }

            // ...

>             // return an integer to set the exit code, or

Reviewer: we don't return anymore
